### ISSUE TITLE
Comments inside frames not syntax highlighted as comments in Beamer syntax

### ISSUE THREAD

**Issue by fredrikpaues** [LaTeXing/LaTeXing#234]
The LaTeXing Beamer syntax does not recognize line comments inside frames as comments. In the code below, the line comment inside the frame environment is highlighted just as a regular line while the comment outside the frame environment is treated as it should be.

    \documentclass{beamer}
    \begin{document}
      % A comment
      \begin{frame}[t]
        \frametitle{title}
        % \framesubtitle{subtitle}
        
      \end{frame}
    \end{document}

**Comment by fredrikpaues** [comment]
On a Windows 7 computer that is synced with the above using [Package Syncing](https://github.com/csch0/SublimeText-Package-Syncing), the above code is formatted as it should be. Strange.

On the Windows 7 machine, the scope is `text.tex.latex.beamer.ing`, `meta.environment.frame.latex`, `comment.line.percentage.tex` while it on the Windows 7 machine is `text.tex.latex.beamer.ing`, `meta.function.environment.frame.latex`.

Any clue what might be wrong?

**Comment by r-stein** [comment]
This is related to #224, Sublime Text Build 3119 has included a new LaTeX syntax. I assume on your Win7 machine has an older ST version.
Since the LaTeXing syntaxes have not been adapted you may just want to use the syntax from `LaTeX/LaTeX.sublime-syntax`, which now has beamer support included.

**Comment by fredrikpaues** [comment]
No, my Win7 machine also has build 3126. But, the default LaTeX works correctly. But LaTeXing changes the syntax back to LaTeXing Beamer on save.

**Comment by r-stein** [comment]
I don't know, but I assume there is a setting to turn that of. You may have extracted the LaTeX package or something like that on one computer and preserved the old syntax.
